src/scripts/rx-data-table.src.js

```diff
@@ -268,7 +272,13 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
                     var classValue = classFunction(row);
 
                     if (!_.isEmpty(classValue)) {
-                        classes[classValue] = true;
+                        if(_.isArray(classValue)) {
+                            _.map(classValue, function (className) {
+                                classes[className] = true;
+                            });
+                        } else {
+                            classes[classValue] = true;
+                        }
```

Comment:
it helps with it, but the true fix for that isn't available till angular 1.4. This does allow for 2 interpolations without ng-class being impacted.